web/src/styles/hub.css

```
@@ -0,0 +1,127 @@
+  --font-display:'Space Grotesk','Inter',ui-sans-serif,system-ui,sans-serif;
+  --font-mono:'JetBrains Mono',ui-monospace,'SF Mono',Menlo,Consolas,monospace;
+  --ground:#ffffff; --panel:#ffffff; --panel-2:#f6f7f9; --ink:#0b0c0e; --muted:#585d67; --subtle:#8b909b;
+  --line:#e7e9ee; --line-2:#d3d7de;
+  --accent:#4f46e5; --accent-2:#4338ca; --accent-tint:#eef0fe;
+  --good:#16a34a; --bad:#e11d48; --warn:#d97706;
+  --radius:9px; --radius-sm:6px;
+  --shadow:0 1px 2px rgba(11,12,14,.04),0 1px 0 rgba(11,12,14,.03);
+  --shadow-lg:0 2px 8px rgba(11,12,14,.06),0 12px 34px rgba(11,12,14,.09);
+  --maxw:940px;
+}
+:root:not([data-theme="light"]){}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)